packages/@okta/vuepress-site/guides/sign-into-mobile-app/get-user-info/android/getinfo.md

```diff
@@ -1 +1,17 @@
-retrieve user info code example
\ No newline at end of file
+An example of getting user information from userinfo endpoint:
+
+```java
+client.getUserProfile(new RequestCallback<JSONObject, AuthorizationException>() {
+    @Override
+    public void onSuccess(@NonNull JSONObject result) {
+        //handle JSONObject result.
+    }
+
+    @Override
+    public void onError(String error, AuthorizationException exception) {
+        //handle error
+    }
+});
+```
+
+In `onSuccess` the userinfo returned is a `JSONObject` with the following [properties](https://developer.okta.com/docs/api/resources/oidc/#response-example-success-5).
```

Comment:
In `onSuccess` the user info returned is a `JSONObject` with the following [properties](https://developer.okta.com/docs/api/resources/oidc/#response-example-success-5).